packages/medusa/src/loaders/admin.ts

```diff
@@ -1,27 +1,27 @@
 import { AdminOptions, ConfigModule } from "@medusajs/types"
+import { ContainerRegistrationKeys } from "@medusajs/utils"
+import { AwilixContainer } from "awilix"
```

Comment:
Is this needed? If it is used somewhere, let's use MedusaContainer instead.

Reply:
I enabled `"noUnusedLocals": true,` in tsconfig and it discovered so many unused imports. Should I keep it on?

Reply:
@thetutlage if we can clean them up easily, let's keep it, otherwise we can leave it for another PR.

Reply:
Cleaned them up and kept it on